### ISSUE TITLE
ARRIBA_VISUALISATION depends on STAR-FUSIOn rather than ARRIBA ?

### ISSUE THREAD

**Issue by HCLbioinfo** [nf-core/rnafusion#493]
### Description of the bug

When launching Arriba only, it seems that Arriba visualisations are not produced. The code suggests this module depends on STAR-fusion rather than Arriba, is it expected ?

https://github.com/nf-core/rnafusion/blob/master/conf/modules.config#L39

### Command used and terminal output

```console
nextflow-23 run
		 -work-dir /srv/scratch/SLURM/19733/work
		 -ansi-log false nf-core/rnafusion
		 -r 3.0.2
		 -c /srv/scratch/DEPOT/pipelines/nf-core/rnafusion_3.0.1/tower.conf
		 -c '"SLURM.conf"'
		 -c '"rnafusion_LS-FFPE-TumeursSolides.conf"'
		 -params-file params_rnafusion_LS-FFPE-TumeursSolides.yaml
		 -profile singularity
		 --input samplesheet.csv
		 --outdir output
		 --read_length 150
		 -resume
```

### Relevant files

`params_rnafusion_LS-FFPE-TumeursSolides.yaml` :
```
arriba:       true
fastp_trim:   true
genomes_base: "/srv/scratch/DEPOT/pipelines/nf-core/rnafusion_3.0.1/references"
```

### System information

- Nextflow 23.10.1 build 5893
- HPC server
- slurm executor
- Singularity container
- Debian 12
- nf-core/rnafusion 3.0.2

**Comment by rannick** [comment]
Indeed, at the moment arriba runs on the output of fusioninspector, which in turn is using the bam files from STAR-fusion. I agree that it is not ideal. It will be changed in the next release related to single mapping

**Comment by nvnieuwk** [comment]
This has been fixed in the `dev` branch and will be available in the next release!
